athena/.env.example

```
@@ -1,2 +1,22 @@
+# SQLite example (relative to each module dir):
+DATABASE_URL=sqlite:///../data/data.sqlite
+# PostgreSQL example:
+# DATABASE_URL=postgresql://user:password@localhost:5432/athena
+
+# OpenAI API configuration
+OPENAI_API_KEY=
+OPENAI_API_VERSION=2023-03-15-preview
+
+# Azure OpenAI API configuration
+AZURE_OPENAI_API_KEY=
+AZURE_OPENAI_ENDPOINT="https://ase-se01.openai.azure.com"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)